Utilities.h

```diff
@@ -1108,7 +1108,7 @@ void eeprom_update(int mapped_addr, uint8_t byte) {
 }
 
 void eeprom_write(uint8_t addr, uint8_t byte) {
-	if (!eeprom_info_locked() && addr >= 0 && addr < EEPROM_RESERVED) {
+	if (!eeprom_info_locked() && addr < EEPROM_RESERVED) {
```

Comment:
No harm here really to also check that addr is > 0. I know it's a uint, but not checking for that rides on the typo implicitly.